boot/storage.txt

```diff
@@ -0,0 +1,285 @@
+    ____________________________________________________________________________
+    
+    This example assumes the target device block of /dev/sda, using gdisk for 
+    the actual partitioning process.
+    
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |    root@localhost ~ # gdisk /dev/sda                                     |
+    |    Create GPT partition table:                                           |
+    |    Command: o                                                            |
+    |    This option deletes all partitions and creates a new protective MBR.  |
+    |    Proceed? (Y/N): y                                                     |
+    |                                                                          |
+    |    Create Partition 1 (/boot):                                           |
+    |    Command: n                                                            |
+    |    Partition Number:  (RETURN key for 1)                                 |
+    |    First sector:                                                         |
+    |    Last sector: +128M                                                    |
+    |    Hex Code:                                                             |
+    |                                                                          |
+    |    Create Partition 2 (UEFI ESP):                                        |
+    |    Command: n                                                            |
+    |    Partition Number:  (RETURN key for 2)                                 |
+    |    First sector:                                                         |
+    |    Last sector: +33M                                                     |
+    |    Hex Code: EF00                                                        |
+    |                                                                          |
+    |    Create Partition 3 (/):                                               |
+    |    Command: n                                                            |
+    |    Partition Number:  (RETURN key for 3)                                 |
+    |    First sector:                                                         |
+    |    Last sector:  (RETURN key for rest of disk)                           |
+    |    Hex Code:                                                             |
+    |                                                                          |
+    |    Write Partition Table To Disk:                                        |
+    |    Command: w                                                            |
+    |    Do you want to proceed? (Y/N): Y                                      |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
```

Comment:
We might want a follow-up section ('post install') that includes a discussion about `/etc/fstab`. A malconfigured `fstab` causes more problems than a missing one - if the `swapfile` gets renamed and `fstab` isn't updated, for instance, the user is dropped into an emergency shell which, while an uneccessary precaution for the system to take, can be a little disconcerting for users and should therefore be explained.

Reply:
related - if we are going to promote a happy-path example, how do we feel about using labels in /etc/fstab